media-fonts/caligula/caligula-1.0.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="A script font"
+DESCRIPTION="Caligula is a crude representation of what was originally a \
+calligraphic bitmap script font autotraced by Fontographer 3.2. There are no \
+kerning pairs in this font and it should probably be used only in sizes 24pt \
+or larger."
+HOMEPAGE="https://github.com/FontCollection/Caligula"
+COPYRIGHT="1994 or earler"
```

Comment:
Still a bit puzzled on the COPYRIGHT here, it's only a date, I'll let other chime in if they have any thoughts on this. Thanks so far!

Reply:
The only reason I made a note of it is that Creative Commons licenses weren't available at the time this design was written.  No author is stated and, apart from the documentation requirement, it was declared Public Domain in the U.S.  Would a relicense be appropriate in such a situation?

Reply:
If the author is unknown, I suppose there is no reason to list the copyright date at all.  Should I remove it altogether?

Reply:
No one stepped up, so guess we can merge this.